### PR TITLE
add cards for higher mass VH

### PR DESCRIPTION
**Pull request by hroskes**
and common cards for JHUGen reweighting for BR(m4l).

The VH mass scan cards for 2016 are 100% identical with the lower mass ones in the pre2017 tag besides the mass and width.  I wanted to keep it this way for simplicity.  For the 2017 ones which will come soon, I'll do the clean up procedure.

### CODE REVIEW

File: bin/Powheg/production/pre2017/13TeV/HWJ_JHUGen_HZZ4L_NNPDF30_13TeV/HWminusJ_HanythingJ_NNPDF30_13TeV_M1000.input [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+withnegweights 1  ! (default 0) if on (1) use negative weights
+#bornonly 1        ! (default 0) if 1 do Born only
+testplots  1      ! (default 0, do not) do NLO and PWHG distributions
+
+#xupbound 2d0   ! increase upper bound for radiation generation
+
+
+iseed    SEED        !  Start the random number generator with seed iseed
+#rand1     0          !  skipping  rand2*100000000+rand1 numbers.
```
Comment (perrozzi) [inline bin/Powheg/production/pre2017/13TeV/HWJ_JHUGen_HZZ4L_NNPDF30_13TeV/HWminusJ_HanythingJ_NNPDF30_13TeV_M1000.input]: please remove commented lines, unless are really needed

**Comment by perrozzi** [comment]
the datacards appear to be exactly the same apart from mass and width.
we recommend uploading one example card and a script to obtain all the others, to reduce the number of files in the repository (which slows down git). would it be a problem?

File: bin/Powheg/production/pre2017/13TeV/HZJ_JHUGen_HZZ2LX_NNPDF30_13TeV/HZJ_HanythingJ_NNPDF30_13TeV_template.input [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+facscfact  1d0   ! (default 1d0) fac scale factor: mufact = muref * facscfact 
+
+! Parameters to allow or not the use of stored data
+use-old-grid    1 ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+
+! the typical call uses 1/1400 seconds (1400 calls per second)
+ncall1 1000000      ! Number of calls for the construction of the importance sampling grid
+itmx1 3           ! Number of iterations for grid
+ncall2 75000      ! Number of calls for the computation of the upper bounding envelope
```
Comment (perrozzi) [inline bin/Powheg/production/pre2017/13TeV/HZJ_JHUGen_HZZ2LX_NNPDF30_13TeV/HZJ_HanythingJ_NNPDF30_13TeV_template.input]: to me this looks like a bit low for ncall2, can you please have a look at the top files for the integration grids?
https://twiki.cern.ch/twiki/bin/viewauth/CMS/PowhegBOXPrecompiledCheckGrids

**Comment by perrozzi** [comment]
apart from the comment (that can be addressed later) the rest looks ok, thanks a lot.
merging

**Comment by hroskes** [comment]
Hi @perrozzi, I copied these cards from the [ones at low mass](https://github.com/cms-sw/genproductions/tree/pre2017/bin/Powheg/production/V2/13TeV/Higgs/HWJ_JHUGen_HZZ4L_NNPDF30_13TeV).  I noticed that ZH had a factor of 10 more ncalls than WH and didn't really understand why, but I think this was all checked by David at the time.

Of course it's possible that something might be different at higher mass.  I ran topdrawer on the 2 TeV one.  Here's the result:
[pwgbtlgrid.ps.gz](https://github.com/cms-sw/genproductions/files/1437908/pwgbtlgrid.ps.gz)

None of them look anything like the bad example on the twiki.  These are a few of the worst ones.  Does it look ok to you?

pwgbtlgrid-16
pwgbtlgrid-19
pwgbtlgrid-26
pwgbtlgrid-5

**Pull request merged**